### DISCUSSION TITLE
Sharing symlinks

### DISCUSSION THREAD

**Discussion by ltzhk65** [nicotine-plus/nicotine-plus#3129] [Q&A]
Hello,

sorry in advance if this has been anwered before. I have qustions regarding sharing symlinks on windows 11. I have read in another discussion that nicotine+ shares files through symlinks (https://github.com/nicotine-plus/nicotine-plus/discussions/2130). My questions:

1. is this true for windows?
2. is this feature always active, or does this have to be configured?
3. is this working for folders as well?

Thanks in advance and for the great work.

**Comment by slook** [comment]
Yes it should work for following file and folder symlinks without any additional configuration, although it is not often tested on your platform. Please keep us posted as to how it goes?

**Reply by ltzhk65** [reply]
Thanks for the reply! I keep watching my uploads, but the linked directories did not come up yet.

This is my structure. I keep adding subdirs for updates.

base_dir
link_to_subdir_1
link_to_subdir_2
   sub_dir_1
   sub_dir_2

My expectation is that anyone who downloads base_dir will get the sub_dirs as well - correct?

**Reply by slook** [reply]
You can see exactly how your shares are presented to downloaders by simply browsing yourself...

Menu -> Browse Shares -> **Browse Public Shares**

All the folders and files that are listed in the Browse Shares interface will also appear in matching searches on the network.

You can test that the client does actually have access to the listed files by right clicking on a file for the context menu and selecting **Open File**. If the file plays/opens in an external program so you can hear/see/read it or whatever, that means it should be available from the downloader's end too.
